src/parser/shared/modules/spells/bfa/essences/TheWellOfExistence.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+import { formatNumber, formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+
+import ItemHealingDone from 'interface/others/ItemHealingDone';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemStatistic from 'interface/statistics/ItemStatistic';
+import StatisticGroup from 'interface/statistics/StatisticGroup';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+
+const debug = false;
+
+// Can these values be queried? 
+const OVERHEAL_ABSORB_RATE = 0.15;
+const RANK_TWO_HEALING_BOOST = 1.5;
+// https://www.wowhead.com/spell=299936/the-well-of-existence
+const RANK_THREE_DOUBLE_CAP = 10982;
```

Comment:
this isn't a constant as it scales with (neck)itemlevel; as its -8 scaling as well we're gonna have to grab out the azerite scales once again. once https://github.com/WoWAnalyzer/WoWAnalyzer/pull/3203 is merged you'll be able to use the code example below to find out the actual cap


`import { calculateAzeriteEffects } from 'common/stats';`
`r3doublecap = calculateAzeriteEffects(296193, this.selectedCombatant.neck.itemLevel)[0]` 
gonna have to create a SPELL for 296193 (for whatever reason everything in azerite has its own spellids, 296193 is the rank 3 minor effect addition)

Reply:
also means that this isn't a magic number anymore as this will be automatically updated whenever azerite scalings get exported

Reply:
Ok. Great, thanks. I'll merge in master once that PR is completed, then update this.